data/loot_lists/level1/level1_accessories.yml

```diff
@@ -5,8 +5,8 @@ generate:
 weighted_entries:
   amulet_gold:
     weight: 1
-    adjective1:

```

Comment:
Thank you for updating all the loot lists to an array format; that is quite a bit of work!